### PR TITLE
User Docs - Getting Started - changes made for up to date info and improvement

### PR DESCRIPTION
**Pull request by LiborBatek**
Improvements and updated info and media.

### CODE REVIEW

File: website/docs/artist_getting_started.md [hunk 1]
```diff
@@ -9,17 +9,19 @@ import TabItem from '@theme/TabItem';
 
 ## Installation
 
-AYON comes in packages for Windows (10 or Server), Mac OS X (Mojave or higher), and Linux distribution (Centos, Ubuntu).
+**AYON** and its installation files are available for all major operating systems: **Windows** 10 (Server, WIN 11 and newer), **macOS** (Mojave or later), and **Linux distributions (Rocky Linux, Ubuntu and formerly CentOS).
+
+Besides getting AYON installation files from Github you also have option to get those via `AYON server` webUI in the `Studio Settings` section.
```
Comment (LiborBatek) [inline website/docs/artist_getting_started.md:14]:
```suggestion
**AYON** and its installation files are available for all major operating systems: **Windows** 10 (Server, WIN 11 and newer), **macOS** (Mojave or later), and **Linux distributions (Rocky Linux, Ubuntu and formerly CentOS).
Besides getting AYON installation files from Github you also have option to get those via `AYON server` webUI in the `Studio Settings` section.
```

File: website/docs/artist_getting_started.md [hunk 2]
```diff
@@ -9,17 +9,19 @@ import TabItem from '@theme/TabItem';
 
 ## Installation
 
-AYON comes in packages for Windows (10 or Server), Mac OS X (Mojave or higher), and Linux distribution (Centos, Ubuntu).
+**AYON** and its installation files are available for all major operating systems: **Windows** 10 (Server, WIN 11 and newer), **macOS** (Mojave or later), and **Linux distributions (Rocky Linux, Ubuntu and formerly CentOS).
```
Comment (LiborBatek) [inline website/docs/artist_getting_started.md:12]:
```suggestion
**AYON** and its installation files are available for all major operating systems: Windows 10 (Server, WIN 11 and newer), **macOS** (Mojave or later), and **Linux distributions (Rocky Linux, Ubuntu and formerly CentOS).
```

Comment (LiborBatek) [reply website/docs/artist_getting_started.md:12]: I don't agree

**Review by tatiana-ynput — APPROVED** [review]
looks good

**Pull request merged**